icecap/ci/run.sh

```diff
@@ -1,5 +1,20 @@
 set -e
```

Comment:
Can you put the standard authorship/licensing information as a comment at the top of this file?

Reply:
Done.